lib/Translations/PotMaker.php

```diff
@@ -46,11 +46,14 @@ public function __construct( Project $config ) {
 	public function make_pot() : void {
 		$source_path = $this->config->get_source_path();
 		$pot_path    = $this->config->get_pot_path();
-		if ( ! is_dir( $source_path ) ) {
-			throw new DirectoryDoesntExist(
-				'Source directory for generating pots does not exist. Path received was:' .
-				" {$source_path}\n"
-			);
+		$sources = is_array( $source_path ) ? $source_path : [ $source_path ];
+		foreach ( $sources as $source ) {
+			if ( ! is_dir( $source ) ) {
+				throw new DirectoryDoesntExist(
+					'Source directory for generating pots does not exist. Path received was:' .
+					" {$source}\n"
+				);
+			}
```

Comment:
Removing complexity.

- 1 negation.
- 1 indentation.

```suggestion
			if ( is_dir( $source ) ) {
				continue;
			}
			throw new DirectoryDoesntExist(
				'Source directory for generating pots does not exist. Path received was:' .
				" {$source}\n"
			);
```